docs/guides/event_loop.md

```diff
@@ -23,29 +85,86 @@ while True:  # infinite loop!
             process_event(event)
 ```
 
-Actions taken by the user add events to the queue ("button pressed",
+Actions taken by the user add events to the queue (e.g. "button pressed",
 "slider moved", etc...), and the event loop handles them one at a time.
 
-## The Qt Event Loop
+## Qt Applications and Event Loops
 
 Currently, napari uses Qt as its GUI backend, and the main loop handling events
-in napari is the
-[Qt EventLoop](https://wiki.qt.io/Threads_Events_QObjects#Events_and_the_event_loop).
-When you use the following syntax:
+in napari is the [Qt
+EventLoop](https://wiki.qt.io/Threads_Events_QObjects#Events_and_the_event_loop).
+
+A deep dive into the Qt event loop is beyond the scope of this document, but
+it's worth being aware of two critical steps in the "lifetime" of a Qt
+Application:
+
+1) Any program that would like to create a
+   [`QWidget`](https://doc.qt.io/qt-5/qwidget.html) (the class from which all
+   napari's graphical elements are subclassed), must create a
+   [`QApplication`](https://doc.qt.io/qt-5/qapplication.html) instance *before*
+   instantiating any widgets.
+
+   ```python
+   from qtpy.QtWidgets import QApplication()
+
+   app = QApplication([])
```

Comment:
Maybe add a comment that `[]` are the args passed to the app or something?

Just, to not leave that hanging in there without an explanation :)